From f09c8008948ae2a3d206af5bf95710d98d6dfafc Mon Sep 17 00:00:00 2001
From: "Rafael J. Wysocki" <rafael.j.wysocki@intel.com>
Date: Fri, 25 Mar 2016 22:33:10 +0100
Subject: [PATCH 1/8] Fix permissions of drivers/power/avs/rockchip-io-domain.c

Git-commit: 8e653b654497350f9cb3d6c2088207bc503f2c7c
Patch-mainline: v4.6-rc1
References: fate#322003

The permissions of this file were modified by commit (f447671b9e4f PM /
AVS: rockchip-io: add io selectors and supplies for rk3399) by mistake,
so fix them.

Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>
Signed-off-by: Mian Yousaf Kaukab <yousaf.kaukab@suse.com>
---
 drivers/power/avs/rockchip-io-domain.c | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 mode change 100755 => 100644 drivers/power/avs/rockchip-io-domain.c

diff --git a/drivers/power/avs/rockchip-io-domain.c b/drivers/power/avs/rockchip-io-domain.c
old mode 100755
new mode 100644
-- 
2.11.0

